### DISCUSSION TITLE
Who wants to be hired (July 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#38459] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #37380. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!* 

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #38458.

**Comment by Manzi-Cedrick** [comment]
Location : USA,Colorado
Remote : Available
Willing to relocate : True
Technologies : Next Js,Typescript,React Js,Tailwind Css,SASS, BootStrap,Node Js,Express Js,BlockChain, Solidarity,Nest Js,Ethereum,UI/UX Design,web development proffesional 
website : http://arbor-fashion.vercel.app/
Email : cedrickmanzii0@gmail.com

**Comment by Phoenix-Alpha** [comment]
Hi,

I am Alex, result-driven software engineer with over 8 years of professional career in startups and looking for full time position.

Location: Bulgaria
Remote: Yes
Willing to relocate: No
Technologies: React, Next, Tailwind CSS, GQL, Hasura, Angular, NgRx, RxJS, AWS Cloud & DevOps
Resume or Website: betafoenix.me
Email: [alexjin0704@gmail.com](mailto:alexjin0704@gmail.com)

Looking forward to talking to you!

Regards

**Reply by Bartesque** [reply]
Hey Alex! Would you be interested in some freelance work?

**Reply by Phoenix-Alpha** [reply]
Yup

**Reply by Phoenix-Alpha** [reply]
Please reach me out via my email

**Reply by HimusakiProgramming** [reply]
@Phoenix-Alpha what your email.?

**Comment by Hurly77** [comment]
HI I'm Cameron J. Leverett,  I have an enthusiastic personality and I love learning.

Location: Anaheim California
Remote: &#128077;
Willing to relocate:👎
Technologies: JavaScript, React, NextJS, Ruby, ROR, Tailwindcss, Cloud, APIs, OOP, Typescript, Postgres, RESTful APIs,
Resume/CV:https://mgqajzspdkingrdcdwwl.supabase.co/storage/v1/object/public/pub-files/blk-crow-ai.pdf#page=1
Website: https://cjleverett.me
Email: camrbo@gmail.com

**Comment by Babbili** [comment]
Hi, I'm a Full Stack Engineer,  I focus towards JavaScript & its frameworks, **Next Js** is my framework of choice, and i'm passionate about experience growth and learning new things.

Location: Dubai, UAE.
Remote: Yes
Willing to relocate: Yes.
Technologies: Next Js, React Js, Node Js, Nest Js, TypeScript, Sass, CSS, GraphQl, MongoDB, Firebase, GCP, Azure, Docker, Kubernetes
Websites: [https://giftandmore.shop](https://giftandmore.shop/), [https://flipcoins.vercel.app](https://flipcoins.vercel.app/), [https://vr-blog.vercel.app/](https://vr-blog.vercel.app/), [more](https://github.com/Babbili?tab=repositories)
Email: [babbili.dxb@gmail.com](mailto:babbili.dxb@gmail.com)

Thank you, contact me for more info.

**Comment by visitskyworld** [comment]
Location: Burnaby, Canada
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, Node.js,Web3.js, Javascript, Typescript, Vue, Nuxt.js, GraphQL, TailwindCSS, ChakraUI, Docker
Website: https://visitskyworld.github.io
Resume: Please check my website 😄
Email: visitskyworld@gmail.com

**Comment by ghost** [comment]
Hello, everyone. I’m a **senior full-stack engineer** with 10+ years of experience in web development, managing all aspects of the development process. I've worked on 70+ projects and solved complex business problems in various industries including marketing, healthcare, eCommerce, real estate, and car insurance. I have consistently saved costs while increasing profits. I am ready for an opportunity to use my skills to grow your revenue exponentially. 

**Location**: Canada
**Remote**: Yes
**Willing to relocate**: No
**Technologies**: 
- languages
   * JS, TS, PHP, Python
- front-end
   * React, Next.js, Vue.js, Nuxt.js, Angular for JS frameworks/libraries
   * Bootstrap, Material UI, Tailwind CSS, Vuetify for UI frameworks
- back-end
  * Express, Laravel for frameworks
  * MySQL, MongoDB, PostgreSQL for database
- services && libraries
  * Mocha, Enzyme, Chai for testing
  * Github, Bitbucket for version control
  * AWS, Netlify, Heroku, Firebase for deployment

**Resume/CV**: [linkedin](https://www.linkedin.com/in/lucas-jiang-486516244/)
**Website**: https://github.com/star-dev-jiang
**Email**: [lucas.jiang.itdev@gmail.com](https://lucas.jiang.itdev@gmail.com)

My goal has always been to exceed the expectations of my clients.
If you have a challenging or interesting project, don't hesitate to contact me.
Thank you for your careful attention.

Best regards.🎉

**Comment by Ali-Hussein-dev** [comment]
### Ali Hussein
Location: Hamburg, Germany
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, TS, JS, Node.js, TailwindCSS, ChakraUI, FaunaDB
Website: https://ali-hussein.vercel.app
Resume: Downloadable from my website :)
Email: [ali.hussein.pre@gmail.com](mailto:ali.hussein.pre@gmail.com)

**Comment by SanjivG10** [comment]
Location: Nepal
Remote: yes
Willing to relocate: no
Technologies:  NextJS, Node.js, Django, Flask, Web Scrapping, Machine Learning, Java, Typescript, AI
Resume/CV: https://www.linkedin.com/in/sanjiv-gautam/
Website: Haven't built yet. 
Email: bhairajagautam@gmail.com

**Comment by crisner1978** [comment]
Location: Murfreesboro, TN
Remote: Yes
Willing to relocate: Nope
Technologies: React JS, Next JS, Node JS, TypeScript, Redux Toolkit, Recoil, MongoDB, Express, Mongoose, Firebase, Sanity, GraphCMS, RESTful APIs, GraphQL, CSS, Bootstrap, Styled Components, TailwindCSS, GIT
Resume/CV: [Resume](https://zety.com/profile/chris-risner)
Website: https://risner-portfolio-app.web.app/
Email: crisner.prsvr@gmail.com

**Reply by Ali-Hussein-dev** [reply]
check your website url, it is not correct

**Reply by crisner1978** [reply]
Corrected it thanks! It works now

**Comment by whayu901** [comment]
Location: Indonesia
Remote: yes
Willing to relocate: no
Technologies: NextJS, React Native, Swift, Javascript, Vuejs, Java, Typescript
Resume/CV: https://www.linkedin.com/in/wahyu-setiawan26/
Website: Haven't built yet.
Email: [cahyonowahyus123@gmail.com](mailto:cahyonowahyus123@gmail.com)

**Reply by s1nny0ur** [reply]
@whayu901 Sent you an email 👌🏻

**Comment by IroncladDev** [comment]
Hello, I'm Conner and am currently looking for a job where I can wield my next.js skills for the better good of the world.

Location: Whitewright, TX
Remote: Yes
Willing to relocate: No
Technologies: Next.js, React, html/css/js, mongodb, graphql, Node/Express, Git/Github, SocketIO
Resume/CV: https://www.connerow.dev/resume.pdf
Website: https://www.connerow.dev
Email: conner@connerow.dev

**Reply by Ali-Hussein-dev** [reply]
I like your website and your skills, wish you all the best!

**Comment by crackwebai1024** [comment]
[FOR HIRE][REMOTE][FULL-STACK prefer FRONTEND]

Remote : Yes
Willing to relocate: No
working timezone : client's timezone (including UK, EST, CST, PST)
Looking for : fully remote job (full-time)
mainskills : 
 - Frontend 
     (SPA - React, Angular, Vue, Svelte)
     (SSR - Next, Nuxt, Sapper)
     (UI - material ui, tailwind CSS, Chakra UI, styled-components, vuetify)
 - Backend (Node - express, Python - django, aws Lambda(chalice, nodejs))
portfolio : http://mingxing.vercel.app/
email: piaobristar@gmail.com

**Comment by ghost** [comment]
Hello, everyone. I’m a **senior full-stack engineer** with 10+ years of experience in web development, managing all aspects of the development process. I've worked on 70+ projects and solved complex business problems in various industries including marketing, healthcare, eCommerce, real estate, and car insurance. I have consistently saved costs while increasing profits. I am ready for an opportunity to use my skills to grow your revenue exponentially. 

**Location**: Canada
**Remote**: Yes
**Willing to relocate**: No
**Technologies**: 
- languages
   * JS/TS, PHP, Python
- front-end
   * React, Next.js, Vue.js, Nuxt.js, Angular for JS frameworks/libraries
   * Bootstrap, Material UI, Tailwind CSS, Vuetify for UI frameworks
- back-end
  * Express, Laravel for frameworks
  * MySQL, MongoDB, PostgreSQL for database
- services && libraries
  * Mocha, Enzyme, Chai for testing
  * Github, Bitbucket for version control
  * AWS, Netlify, Heroku, Firebase for deployment

**Resume**: [linkedin](https://www.linkedin.com/in/lucas-jiang-486516244/)
**Website**: https://github.com/star-dev-jiang
**Email**: [lucas.jiang.itdev@gmail.com](https://lucas.jiang.itdev@gmail.com)

My goal has always been to exceed the expectations of my clients.
If you have a challenging or interesting project, don't hesitate to contact me.
Thank you for your careful attention.

Best regards.🎉

**Comment by shahidul7150** [comment]
Hello, I'm Shahidul Islam from Bangladesh and am currently looking for an intern where I can wield my MERN stack skills for the better good of the world.
Remote: Yes
Willing to relocate: No

Looking for: fully remote intern (full-time)

mainskills :
React, Next,Redux,Node - express,Mongodb, tailwind CSS,Bootstrap5

portfolio : https://prog-shahid.web.app/
email: shahidulislam1474@gmail.com
[Resume of SHAHIDUL ISLAM .pdf](https://github.com/vercel/next.js/files/9125134/Resume.of.SHAHIDUL.ISLAM.pdf)

**Comment by DujeUsalj** [comment]
Location : Croatia
Remote : YES
Willing to relocate : NO
Technologies : Next Js,Typescript,React Js,Tailwind CSS, Firebase, Next Auth
Email : duje.usalj@gmail.com

**Comment by jeffrey125** [comment]
Location: Philippines
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js, TypeScript, tailwindcss, React, Express, MongoDB, NodeJS, Firebase.
Resume/CV: [Resume](https://jeffreyasilo.com/resume)
Website: [My Website](https://jeffreyasilo.com/)
Email: wd@jeffreyasilo.com

**Comment by GreatWorldDev** [comment]
Location: Canada
Remote: Yes
Website: https://linkedin.com/in/hayatosuzuki801293
Email: hayato.suzuki.boy@gmail.com

🌻 Open to work 🌻 

[FOR HIRE][REMOTE] [FULL STACK] [BLOCKCHAIN] [FULL/PART TIME] [ANY TIMEZONE]

Leading the business to succeed is my first goal and responsibility. 
Seeing is believing. Trust you, trust me.

I am a Front-end developer who builds Well-Responsive, Pixel-Perfect, functional, beautiful, dynamic Websites.
I always keep clean, tidy, easily readable, and maintainable code for delivery and this always helped me to succeed in every job.

Here are my skill sets.
E-Commerce, Healthcare, Education, KYC, Functional Dashboard, Dapp, DEFI, NFT, Finance Websites.

🌻   [FRONT END]
★ React, Next.js, Redux, React Hooks, Material UI, Semantic UI, Vue, Nuxt.js, Vuetify 
★ Tailwind CSS, Bootstrap, Owl Carousel
★ ES6+, TypeScript
★ SASS, Styled-Components
★ D3.js, Immutable.js, WebGL

🌻 BONUS

 🔸  [BLOCKCHAIN]
  ★ Ethereum/Polygon/Solana/Algorand/Rust/Defi/Solidity/Web3.js/Ethers.js
 🔸  [BACK END]
  ★ Node.js, Express, Nest, Laravel, Python, Golang
  🔸 [DATABASE]
  ★ MongoDB, MySQL, PostgreSQL, GraphQL, Firebase, Redis

Just let me know if you want to build any type of website. 
Hope to see you soon.
Regards. 😎

**Comment by abhiram11** [comment]
Location: India
Remote: Available and preferred!
Willing to relocate: Negotiable
Technologies: NextJS, React, React Native, Python, JavaScript, Tensorflow, Github Pages and Actions, Firebase.
Resume/CV: 
[Abhiram Satpute Resume.pdf](https://github.com/vercel/next.js/files/9130491/Abhiram.Satpute.Resume.pdf)
Website: https://abhiramsatpute.com
Email: abhiramsatpute@gmail.com

Other Social Profiles:
Stack Overflow: https://stackoverflow.com/users/8185479/abhiram-satput%c3%a9 (Reputation: 586, Impact: ~96,000 people reached, 1 Gold, 4 Silver, 20 Bronze Badges)
Blogs: https://abhiramsatpute.hashnode.dev/ (Blogs on Docker, AWS, React (+ Native, Query, etc.)
Youtube: https://www.youtube.com/channel/UCfk8qjnhHsVTuygD9fwlx7g

**Reply by abhiram11** [reply]
I hope I'm not too late for the party!

**Comment by ghost** [comment]
Hello, everyone. I’m a **senior full-stack engineer** with over a decade of experience in web development, managing all aspects of the development process. I've worked on tons of small and big projects and solved complex business problems in various industries including marketing, healthcare, eCommerce, real estate, and car insurance. I have consistently saved costs while increasing profits. I am ready for an opportunity to use my skills to grow your revenue exponentially. 

**Location**: Canada
**Remote**: Yes
**Willing to relocate**: No
**Technologies**: 
- languages
   * JS/TS, PHP, Python
- front-end
   * React, Next.js, Vue.js, Nuxt.js, Angular for JS frameworks/libraries
   * Bootstrap, Material UI, Tailwind CSS, Vuetify for UI frameworks
- back-end
  * Express, Laravel for frameworks
  * MySQL, MongoDB, PostgreSQL for database
- services && libraries
  * Mocha, Enzyme, Chai for testing
  * Github, Bitbucket for version control
  * AWS, Netlify, Heroku, Firebase for deployment

**Resume**: [linkedin](https://www.linkedin.com/in/lucas-jiang-486516244/)
**Website**: https://github.com/star-dev-jiang
**Email**: [lucas.jiang.itdev@gmail.com](https://lucas.jiang.itdev@gmail.com)

My goal has always been to exceed the expectations of my clients.
If you have a challenging or interesting project, don't hesitate to contact me.
Thank you for your careful attention.

Best regards.🎉

**Comment by 2manoj1** [comment]
Location: Bangalore India
Remote: Yes
Willing to relocate: Yes
Technologies: Nextjs, Reactjs React Native, Fastifyjs, Prisma, Postgress
Resume/CV: https://drive.google.com/file/d/1MNMJdycbHO35buEiHRaYFU4lDEFFSNI1/view?usp=drivesdk
Website: LinkedIn - https://www.linkedin.com/in/manoj-mukherjee
Email: manojmukherjee777@gmail.com

**Comment by kellenwiltshire** [comment]
Hi Everyone,

My name is Kellen Wiltshire and I am a passionate full-stack developer specializing in all aspects of Web Development looking for the next step in my journey. I have the experience to create beautiful and functional websites combining the latest frameworks and technologies. I have experience in React(NextJS), Javascript/Typescript, TailwindCSS, Styled Components, as well as traditional HTML5 and CSS3. I am currently look for a new Web Development career opportunity! If you are willing to give me the opportunity I promise you won't find someone more motivated to exceed expectations than me!

Location: Ontario, Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Emotion, Typescript, Squareup/Shopify
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Comment by sathwikram** [comment]
Hi, Everyone

I am a senior full-stack engineer with 6+ years of experience in web and mobile app development. 

Location: India
Remote: Yes
Willing to relocate: No
Technologies: React, React Native, Next Js, Express Js, Node, Hasura, PostgresSQL, Typescript
Resume/CV: [ram_kumar_resume](https://1drv.ms/w/s!AhzBROm703RGlHGt9ns8836zAp_D?e=iRAJhT)
Website: https://www.linkedin.com/in/sathwikram/
Email:sathwikram@outlook.com

**Comment by Shofol** [comment]
Hi, This is Shofol. 

I'm a Frontend developer with experience in building market-ready production level web applications from scratch. I'm working professionally for more than 4 years. Currently I'm looking for freelance/project based works to hop in. 

Location: Dhaka, Bangladesh
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, TypeScript, React.js, Next.js, Angular 2+, HTML5, CSS, SCSS, TailwindCSS, Bootstrap, etc. 
Resume/CV: https://drive.google.com/file/d/1zvnek7eLexWCyhMtH6KhYptF0cZq6zuy/view?usp=sharing
Website: https://github.com/Shofol
Email: jahananower@gmail.com

**Comment by allImran** [comment]
[FOR HIRE] [REMOTE] [Part-Time / Full-Time / Contract] [FRONT END DEVELOPMENT] [FULL STACK DEVELOPMENT]
Hello. 
This is Shuvo, I am from Bangladesh.
I'm a senior full stack web developer for 5+ years of experiences. Passionate about pushing the tech industry and new idea forward by working with my skills and experience. I always love to learn new cutting edge technologies and overcome challenges. Available 40 hours per week, fully remote in several time zones.

Linkedin: https://www.linkedin.com/in/sar333/
Github: https://github.com/neutral-Stage
Personal Website: http://shuvoanirbanroy.com/

❤️‍🔥 Special Expert
- Any popular headless CMS such as Sanity, Strapi, Prismic, GraphCMS
- Specialised in Shopify Custom Storefront using (Nuxt/Vue/ React/ Gatsby/ Next Js) 
- Any Custom Map( Google Map/ Mapbox)
- School/College/ Hotel Management System
- Solving Complex Problem
- Chat Application
- Mobile App
- SMS api backend using Twilio
- Awesome Animation (Framer Motion, GSAP, Anime, Three Js)
- Full Dynamic Responsive Website with SEO

🚧 Backend-Development
- Server-Side: PHP/Laravel, Node.js/ Express.js
- Database: MySQL, MongoDB, Firebase
- Headless CMS: Sanity.io, Strapi 

⛓️ Blockchain Development
- Web3, Ethers, Solana ,Solidity
- Moralis, ThirdWeb, Hardhat, Gun, Ganache, MetaMusk, Truffle, Alchemy, D3, Openzeppelin
- NFT token, NFT marketplace
- DeFi, Dapp

🏜️ Front - end development 
- Vue.js/ React.js
- Vuetify/ Bootstrap/ Tailwind CSS
- CSS3/SASS/SCSS/LESS
-Javascript/ ES6 / JQuery
- Gatsby / Nuxt / Next Js

📱 Mobile Development: React Native(Android & iOS)

Do not hesitate to send me a private message directly or by email : anirbanroy691@gmail.com
Ready to start work immediately!

**Comment by eric-burel** [comment]
Hi fellow recruiters and entrepreneurs :vulcan_salute: ,

I am a seasoned JavaScript engineer with experiences in building startup products with Next.js, Electron, GraphQL, Meteor...
especially in the field of Big Data visualization and exploration.

My latest work is moving the Vulcan open-source project to a modern "Monorepository".

This architecture is excellent if your company must manage multiple JavaScript applications that may share some common code.

:man_technologist:  Demo here: https://github.com/VulcanJS/vulcan-npm

I can help you moving your company to a similar Monorepo architecture (among other things :)).

My hourly rate: 120$. I am located in France and can work in French or English speaking environments.

You can reach me out via mail: eb@lbke.fr

Looking forward to working with you!

Location: France
Remote: Yes
Willing to relocate: No (I am a freelance)
Technologies: Next.js, Remix, GraphQL, React, Python, Electron
Resume/CV: My publications: https://medium.com/@eric.burel ; my GitHub with OSS projects:  https://github.com/eric-burel and my previous freelance experiences (in French :fr:) https://www.lbke.fr/eric-burel
Website: https://www.lbke.fr
Email: eb@lbke.fr

**Comment by kimo2121** [comment]
Location: Egypt
Remote: yes
Willing to relocate: yes
Technologies: Next, React, Redux, JavaScript, Typescript, Solidity, HTML5 , CSS3 , Sass, Bootstrap, Material-UI, Semantic-UI, GraphQL, Hasura, Airtable, NodeJs, Firebase, Netlify, Vercel, Heroku, Git, VScode
Resume/CV: https://drive.google.com/file/d/1bx_2mJzWZVyZMQcy4FPmeMVGxbBiX2rz/view?usp=sharing
Website: https://elegant-faun-f98746.netlify.app/
Email: kareem.gaad@gmail.com

**Comment by Ibeandyson** [comment]
Location: Nigeria 
Remote: yes
Willing to relocate: yes
Technologies: Nextjs, React, React Native, Nodejs, Express.js, MongoDB, MySqlDB,   Redux, JavaScript, Typescript,  HTML5 , CSS3 , Material-UI, GraphQL, Firebase,Git,
Resume: https://drive.google.com/file/d/1k3cocYBPSYzpQChwETuwxJTdzdi1fC5U/view?usp=sharing
Linkedin: https://www.linkedin.com/in/ibe-andyson-b290ba176
Email: [ibeandyson123@gmail.com](beandyson123@gmail.com)

**Comment by AstrayMS** [comment]
Location: USA, California
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, SACC / SCSS, Typescript, NodeJS, ExpressJS, PGSQL, MySQL, MongoDB, Handlebars, EJS, Pug, HTML, CSS
Resume/CV: https://drive.google.com/file/d/16rBDpX0bQwI5O9OmkcIa_a23sbekCTYu/view?usp=sharing
Email: saberwatch@gmail.com

**Comment by Izarothy** [comment]
Hi, this is Michał. 

I'm a front-end developer with no previous commercial experience trying to get into the IT world. I'm quite certain that my skills are more than enough for an entry-level position. In any case, I'm available for additional questions or challenges. 

Location: Poland
Remote: Yes
Willing to relocate: No
Technologies: Next, React, Tailwind, Redux, Typescript, Jest, Next-auth, React Hook Form, MongoDB, Prisma
Resume/CV: I will send via mail to any interested interviewers
Website: [https://izaroth.vercel.app]( https://izaroth.vercel.app)
Email: izarothmail@gmail.com

**Comment by ghost** [comment]
Hello, everyone. I’m a **senior full-stack engineer** with over a decade of experience in web development, managing all aspects of the development process. I've worked on tons of small and big projects and solved complex business problems in various industries including marketing, healthcare, eCommerce, real estate, and car insurance. I have consistently saved costs while increasing profits. I am ready for an opportunity to use my skills to grow your revenue exponentially. 

**Location**: Canada
**Remote**: Yes
**Willing to relocate**: No
**Technologies**: 
- languages
   * JS/TS, PHP, Python
- front-end
   * _React, Next.js_, Vue.js, Nuxt.js, Angular for JS frameworks/libraries
   * Bootstrap, Material UI, _Tailwind CSS_, Vuetify for UI frameworks
- back-end
  * Express, Laravel for frameworks
  * MySQL, MongoDB, PostgreSQL for database
- services && libraries
  * Mocha, Enzyme, Chai for testing
  * Github, Bitbucket for version control
  * AWS, Netlify, Heroku, Firebase for deployment

**Portfolio**: https://lucasjiang.netlify.app/
**Linkedin**: https://www.linkedin.com/in/lucas-jiang-486516244/
**Email**: [lucas.jiang.itdev@gmail.com](https://lucas.jiang.itdev@gmail.com)
**Github**: https://github.com/borealia/ 

My goal has always been to exceed the expectations of my clients.
If you have a challenging or interesting project, don't hesitate to contact me.
Thank you for your careful attention.

Best regards.🎉

**Comment by Christian-pprogrammer** [comment]
Location:Rwanda
Remote:Yes
Willing to relocate:No
Technologies:Next js, React js, Node js, Angular, Vue, Java, Springboot, Nest js, AWS, Docker, Kubernetes
Resume/CV:https://www.mpanoc.me/files/resume.pdf
Website:https://www.mpanoc.me
Email:mpanoc6@gmail.com

**Comment by zrothauser** [comment]
Hi everyone, I'm Zack, and I work as a freelance/contract full-stack web developer specializing in JavaScript and React applications. I work with clients who need the technical skills of an experienced front-end developer to create projects that are innovative, maintainable, and easy-to-use. 

Some of my recent projects have included building the front- and back-end of a tool for a market research company, CMS migrations for museums, and a full website build for a venture studio. In the past, I've worked with a number of large publishing companies as well. 

I'm currently looking for part-time or contract opportunities, and not full-time work. 

**Location:** Austin, TX, US
**Remote:** Yes (or possibly on-site in central Texas)
**Willing to relocate:** No
**Technologies:** Next.js, React, PHP, node, REST APIs, Strapi
**Resume/CV:** [https://zackrothauser.com/experience](https://zackrothauser.com/experience)
**Website:** [https://zackrothauser.com](https://zackrothauser.com)
**Email:** zrothauser@gmail.com

**Comment by amirardalan** [comment]
I'm a Front-end Developer with 10+ years experience building outstanding digital products for top brands. I'm currently open to contract or full-time React/Next.js work.

I possess extensive design experience in addition to full stack engineering skills. I'm especially passionate about User Interface and Front-end. I lean toward functional programming paradigms and build my React projects using TypeScript and hooks. I eat CSS for breakfast.

**Location:** Portland, OR, USA
**Remote:** Yes
**Willing to relocate:** No
**Technologies:** TypeScript, React, Next.js, Emotion, Adobe XD, SWR, PostgreSQL, Vercel
**Resume/CV:** https://amirardalan.com/amir-ardalan-resume.pdf
**Website:** https://amirardalan.com
**Github:** https://github.com/amirardalan
**Email:** [hi@amirardalan.com](mailto:hi@amirardalan.com)

**Comment by ghost** [comment]
Hi Amir,

Thank you for reaching out to me.
I am a senior full stack & blockchain developer with 10 years of experience
web & mobile development.

Do you have any prepositions?

Lucas

**Comment by Vokuc** [comment]
Location: Imo, Nigeria
Remote: True
Willing to relocate: True
Technologies: Reactjs, Tailwind, Next
Email: ukpaival@gmail.com
Resume/CV: [CHIKA-UKPAI(4).pdf](https://github.com/vercel/next.js/files/9258600/CHIKA-UKPAI.4.pdf)

**Comment by KhoiUna** [comment]
Location: Florence, Alabama, US.
Remote: Yes
Willing to relocate: Yes (USA)
Technologies: NextJS, ReactJS, NodeJS, Express, TailwindCSS, Material UI, PostgreSQL, MySQL, MongoDB. 
Resume/CV: https://drive.google.com/file/d/1tE5qm-1vYVCJikUfO2YtYjcpS-Mz11Mx/view?usp=drivesdk
Website: www.khoiuna.info
Email: nguyentuankhoi2112@gmail.com

_**Note:** I am an international student studying in the US. I can work for a year after I graduate in May 2023._

**Comment by anewman15** [comment]
Hi,

I'm a Fullstack Rails / React Developer looking for a change. Recently, I've been working with Next.js as a side project. I'm looking for a globally remote role. Fullstack or Frontend, both welcome.

Location: Bangladesh
Remote: Yes
Willing to relocate: No
Technologies: Ruby on Rails, React, Redux, Next.js, GraphQL, MySQL, PostgreSQL, Bootstrap, TailwindCSS
Résumé/CV: https://docs.google.com/document/d/1VxmSmHcTjyzxrVOyRckD6bVcmfKL1aXZZov7g3Dfo2o/edit?usp=sharing
Website: https://anewman15.github.io/
Email: [anewman15@hotmail.com](mailto:anewman15@hotmail.com)

Thanks!

**Comment by cymkd** [comment]
- 10+ years of experience in the digital / startup world
- Experience in range of roles: leadership - management - production/development
- Full stack developer: Nodejs, Reactjs, (Next,Svelte,Astro et al.), Threejs/R3Fiber, Elixir/Pheonix
- Full stack designer: Figma, Framer, Illustrator, Photoshop, After Effect, Blender etc..
- Facilitating experience: Google Design Sprint, Design Thinking
- Industries worked in: FMCG (food & pharma), cosmetics & skincare, air travel, creative industry etc.
- Supporting knowledge: Behavioral psychology, Business planning & consulting, Brand & Relationships marketing
- Languages: English, French, Serbian, Spanish, Chinese.

Looking for full-time or project based work starting second half of September.

Contact: v.simakovic@hey.com

**Comment by bekim-dev** [comment]
Hi, my name is Bekim, highly innovative and motivated Front-end developer, looking for new opportunity to grow and learn new stuff.
Experienced both in web and mobile development but with higher proficiency in web with Next.js and React.js.
I would prefer to work on a small team or a startup with their own product.
Available for full-time, part-time or hourly...

Location: North Macedonia
Remote: Yes
Willing to relocate: Maybe
Technologies: **Next.js**, **React.js**, **JavaScript**, **TypeScript** , **React-Native**, **Node.js**
Resume/CV: [Resume](https://drive.google.com/file/d/14Oh670UY6smVO8b2T_seoC7PJhlXe8Th/view?usp=sharing)
Website: [throwsms.com](https://throwsms.com) (My best work)
Email: bekim.developer@gmail.com

**Comment by soogoodadnan** [comment]
Hi,

I am Adnan, a result-driven software engineer with over 6 years of professional experience in startups seeking a full-time opportunity.

Location: Pakistan
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next, Tailwind CSS, GQL,MongoDb, AWS, NodeJS, Android , Angular, NgRx, RxJS, AWS Cloud & DevOps
Resume or Website: adnanjaved.me
Email: [adnanjaved9598@gmail.com](mailto:adnanjaved9598@gmail.com)

Looking forward to talking to you!

Regards

**Comment by subhampanja2** [comment]
Location : India
Remote: Available
Willing to relocate: No
Technologies  : Next Js,Typescript,React Js,Redux,Tailwind, Material UI, Ant Design, Node JS, Web socket, JWT, MongoDB, Firebase, AWS, GCP
Experience:  Around 4 Years
website : https://subhampanja-28.web.app
Email : [subbhampanja28@gmail.com](mailto:subhampanja28@gmail.com)
